python3/cloudwatch/README.md

```diff
@@ -116,6 +116,8 @@ Give your logs some time to get from your system to ours, and then open [Kibana]
 If you still don't see your logs, see [log shipping troubleshooting](https://docs.logz.io/user-guide/log-shipping/log-shipping-troubleshooting.html).
 
 ## Changelog:
+- **1.0.1**:
```

Comment:
We're adding a new ability, so I think it should be 1.1.0.
Also, let's specify in the changelog that we're now supporting Lambda Insights format.